packages/node/src/utils/node.config.spec.ts

```diff
@@ -1,15 +1,15 @@
 import { join } from 'path';
+import { getNodeWebpackConfig } from './node.config';
+import TsConfigPathsPlugin from 'tsconfig-paths-webpack-plugin';
+import { BuildNodeBuilderOptions } from './types';
+
```

Comment:
Doesn't this change nullify the mocks done below?
AFAIK jest still cannot mock with `import`s, so you have to import/require after the mocks.

Any reason this is not true here?

Reply:
Jest should hoist the mocks iirc (see https://jestjs.io/docs/manual-mocks#using-with-es-module-imports)

Reply:
Like @AgentEnder mentioned, Jest hoists the mocks for you.